test/test_helper.rb

```diff
@@ -118,9 +130,16 @@ class IterationUnitTest < ActiveSupport::TestCase
   end
 
   def insert_fixtures
-    10.times do |n|
-      Product.create!(name: "lipstick #{n}")
-    end
+    now = Time.now
+    products = 10.times.map { |n| { name: "lipstick #{n}", created_at: now - n, updated_at: now - n } }
+    Product.insert_all!(products)
+
+    comments = Product.order(:id).limit(3).map.with_index do |product, index|
+      comments_count = index + 1
+      comments_count.times.map { |n| { content: "#{product.name} comment ##{n}", product_id: product.id } }
+    end.flatten
+
+    Comment.insert_all!(comments)
   end
 
   def truncate_fixtures
```

Comment:
I think you want to change the line below to `ActiveRecord::Base.connection.truncate(Product.table_name, Comment.table_name)`